src/platforms/javascript/common/session-replay/privacy.mdx

```diff
@@ -44,6 +44,12 @@ The following options can be configured as options to the integration, in `new R
 | unblock       | string[]                 | `.sentry-unblock, [data-sentry-unblock]` | Don't redact any elements that match the DOM selectors. Used to unblock specific media elements that are blocked with `blockAllMedia`.                                          |
 | unmask        | string[]                 | `.sentry-unmask, [data-sentry-unmask]`   | Unmask all elements that match the given DOM selectors. Used to unmask specific elements that are masked with `maskAllText`.                                                    |
 
+# Network Request and Response bodies and headers
+
+<Include name="replay-network-sensitive-data.mdx" />
```

Comment:
We have the identical "Sensitive Data in Request and Response Bodies" text in both the Configuration page and Privacy page. I don't know if this is necessary and seems pretty redundant. Especially because we have a Privacy heading under the Configuration page that directs folks to the Privacy page. We also mention why we have an opt-in approach on our "Network Details" paragraph in the Configuration page.

I would remove the paragraph on "Sensitive Data in Request and Response Bodies" from the Configuration page, and only have it on the Privacy page.

Reply:
removing